custom/webapp/imprint.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <!--
+               ____       _   _           _
+              / ___|     | \ | | ___   __| | ___
+             | |  _ _____|  \| |/ _ \ / _` |/ _ \
+             | |_| |_____| |\  | (_) | (_| |  __/
+              \____|     |_| \_|\___/ \__,_|\___|
+
+              Brought to you by the German Neuroinformatics Node (G-Node) @gnode
+              Copyright (c) 2020 G-Node
+              License: BSD-4 clause
```

Comment:
Noticed the copyright info on the pages says BSD-4 clause, but the repo license is BSD-3 clause.  Should they match?

Reply:
Good call, I think they should...